test/react-native/features/override_unhandled.feature

```diff
@@ -2,9 +2,10 @@ Feature: Overriding unhandled state
 
 Scenario: Non-fatal error overridden to unhandled
     When I run "HandledOverrideJsErrorScenario"
-    Then I wait to receive 2 requests
-    And I discard the oldest request
-    And the exception "message" equals "HandledOverrideJsErrorScenario"
+    Then I wait to receive an error
+    And I wait to receive a session
```

Comment:
Is it important to these two scenarios that a session is received - it was simply discarded before, presumably because it was just the first out and needed to be got past.  I suspect we can simply remove the steps to receive a Session here.

Reply:
I think it's reasonable to expect it, as we're verifying the session elements in the error payload and if that request _isn't_ received it's indicative that those elements won't be present because the session wasn't delivered.